modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -7,7 +7,32 @@ class EvidenceWaiverController < ClaimsApi::V2::ApplicationController
         before_action :verify_access!
 
         def submit
-          render json: { status: 'OK' }
+          bgs_claim = find_bgs_claim!(claim_id: params[:claimId])
+
+          raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Claim not found') if bgs_claim.blank?
```

Comment:
if we pass in a random claim ID, it looks like it just gives us a "claim" with a `nil` value. 

```
{:bnft_claim_dto=>nil, :"@xmlns:ns0"=>"http://benefits/"}
```

therefore, we won't hit line 12